financialaid/factories.py

```diff
@@ -43,13 +43,61 @@ class TierProgramFactory(DjangoModelFactory):
     """
     program = SubFactory(ProgramFactory)
     tier = SubFactory(TierFactory)
-    discount_amount = FuzzyInteger(low=1, high=12345)
+    discount_amount = FuzzyInteger(low=50, high=200)
     current = Faker('boolean')
-    income_threshold = FuzzyInteger(low=1, high=10000)
+    income_threshold = FuzzyInteger(low=50, high=200)
 
     class Meta:
         model = TierProgram
 
+    @classmethod
+    def create_properly_configured_batch(cls, num_to_create, tier_size=5000, discount_threshold_pct=80, **kwargs):
+        """
+        Creates a batch of TierPrograms that obey a few constraints:
+        (1) Discount amounts decrease as income increases
+        (2) Discount amounts will not exceed the program price
+        (3) One TierProgram in the batch will have a 0 discount amount, and one will have a 0 income threshold
+        """
+        if num_to_create < 1:
+            raise ValueError('Number of tiers to create must be greater than 0')
+        elif num_to_create == 1:
```

Comment:
I'm not sure we need this check. If you change the denominator for `discount_increment` to not cause a division by zero, everything else should be fine as is